bundle/manifests/external-dns_rbac.authorization.k8s.io_v1_clusterrolebinding.yaml

```diff
@@ -11,3 +11,6 @@ subjects:
 - kind: Group
   name: system:serviceaccounts:external-dns
   namespace: external-dns
+- kind: Group
+  name: system:serviceaccounts:external-dns-operator
```

Comment:
I have a doubt as to why the operator serviceAccount also needs this clusterrolebinding

Reply:
Yes, the operator doesn't need it. It's just that there is no way to exclude 1 service account from the namespace group.
Unfortunately there is not much choice to work this around without giving the operator the update permission on `clusterrolebindings` resource.